lib/bindings/kvbm/python/kvbm/trtllm_integration/consolidator_config.py

```
@@ -8,21 +8,27 @@
 import logging
 import os
 
+from kvbm.utils import get_consolidator_mode, is_truthy
+
+__all__ = [
+    "get_consolidator_mode",
+    "is_truthy",
+    "should_enable_consolidator",
+    "get_consolidator_endpoints",
+]
```

Ruff (0.15.10)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)